src/Castle.Facilities.AspNetCore.Tests/Resolvers/FrameworkDependencyResolverTestCase.cs

```diff
@@ -44,6 +44,12 @@ public void TearDown()
 			testContext.Dispose();
 		}
 
+		[Test]
+		public void Should_not_match_null()
+		{
+			Assert.That(!frameworkDependencyResolver.HasMatchingType(null), Is.True);
```

Comment:
Is there a reason you have `!` and `Is.True`, rather than just using `Is.False`? At first I read this unit as wrong until I saw that not.

Reply:
Dunno. Corrected now.